$NetBSD: patch-aa,v 1.10 2012/07/30 02:53:57 dholland Exp $

- Use optimization level that actually exists (gcc) or works (clang).
- Don't use opengl. (is this still correct?)
- Configure paths for pkgsrc.

--- Imakefile.orig	2002-12-03 17:25:43.000000000 +0000
+++ Imakefile
@@ -2,13 +2,13 @@
 
 /* Fairly important stuff */
 
-#define OptimizationLevel -O4
+#define OptimizationLevel -O2
 #define DoWall NO
 
 #define BuildShared YES
-#define StaticLink YES
+#define StaticLink NO
 
-#define BuildGL YES
+#define BuildGL NO
 
 
 /* Less Important or System Specific */
@@ -33,22 +33,22 @@
 /* Out in the Tall Weeds */
 
 #define InstallNonstandard NO
-#define XFormsNonstandardInstallDir /usr/local
+#define XFormsNonstandardInstallDir $(PREFIX)
 
 #define NonstandardXpm NO
 
-#define XpmIncLoc /usr/local/include
-#define XpmLibLoc /usr/local/lib
+#define XpmIncLoc ${LOCALBASE}/include
+#define XpmLibLoc ${LOCALBASE}/lib
 
 #define NonstandardTiff NO
 
-#define TiffIncLoc /usr/local/include
-#define TiffLibLoc /usr/local/lib
+#define TiffIncLoc ${LOCALBASE}/include
+#define TiffLibLoc ${LOCALBASE}/lib
 
 #define NonstandardJpeg NO
 
-#define JpegIncLoc /usr/local/include
-#define JpegLibLoc /usr/local/lib
+#define JpegIncLoc ${LOCALBASE}/include
+#define JpegLibLoc ${LOCALBASE}/lib
 
 ######################################################################
 #                                                                    #
